### ISSUE TITLE
Results of experiment

### ISSUE THREAD

**Issue by ZERO2ER0** [ZERO2ER0/opticalCNN#4]
# 非线性激活，平铺卷积，quickdraw16，learning_rate=0.0005, num_iters = 10001, DoMean = False
1. train accuracy 0.76, test accuracy 0.7375
# 线性激活，平铺卷积，quickdraw16，learning_rate=0.0005, num_iters = 10001, DoMean = False
1. train accuracy 0.84, test accuracy 0.74625
# 线性激活，平铺卷积，quickdraw16，learning_rate=0.0005, num_iters = 10001, DoMean = Ture
1. train accuracy 0.52, test accuracy 0.441875
# 线性激活，平铺卷积，quickdraw16，learning_rate=0.0005, num_iters = 10001, FC全连接层
# 非线性激活，平铺卷积，quickdraw16，learning_rate=0.0005, num_iters = 10001, FC全连接层
